bin/metrics-elb.rb

```diff
@@ -52,7 +52,7 @@ class ELBMetrics < Sensu::Plugin::Metric::CLI::Graphite
          description: 'Metric naming scheme, text to prepend to metric',
          short: '-s SCHEME',
          long: '--scheme SCHEME',
-         default: ''
+         default: 'loadbalancer'
 
   option :fetch_age,
```

Comment:
After reading through the code it feels to me like `fetch_age` should be renamed `start_time` does that sound right? If so we are already doing lots of breaking changes and might as well make it make more sense.

Reply:
See comment below